src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int upperSide;
+    private int downSide;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, int upperSide, int downSide, int height) {
+        this.color = color;
+        this.upperSide = upperSide;
+        this.downSide = downSide;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (upperSide + downSide) / 2 * height;
```

Comment:
When performing arithmetic operations, especially those involving division and multiplication, it's a good practice to use parentheses to ensure the order of operations is clear and correct. In this case, it would be advisable to enclose the addition operation in parentheses before the division, like this: `return ((upperSide + downSide) / 2.0) * height;`. Also, note that division of integers in Java results in an integer. To get a precise result, one of the operands should be a floating-point number (e.g., `2.0`).